content/docs/tasks/telemetry/kiali/index.md

```diff
@@ -0,0 +1,363 @@
+---
+title: Visualizing Your Mesh
+description: This task shows you how to visualize your services within an Istio mesh.
+weight: 49
+keywords: [telemetry,visualization]
+---
+
+This task shows you how to visualize different aspects of your Istio mesh.
+
+As part of this task, you will install the [Kiali](https://www.kiali.io) add-on
+and use the web-based graphical user interface for viewing service graphs of
+the mesh as well as your Istio configuration objects. You will also use the
+Kiali Public API to generate graph data in the form of consumable JSON.
+
+The [Bookinfo](/docs/examples/bookinfo/) sample application is used as the
+example application throughout this task.
+
+## Before you begin
+
+You first need to create a secret in your Istio namespace with credentials that
+users will use to authenticate to Kiali. See the
+[Helm README](https://github.com/istio/istio/blob/master/install/kubernetes/helm/istio/README.md#installing-the-chart)
+for details, but here is a command you can run to create a secret:
```

Comment:
...here is an example command you can modify and run....   (Just so people don't blindly copy and paste without adding their own configuration data)

Reply:
this is done.